src/renderer/ScreenController.tsx

```diff
@@ -161,6 +162,12 @@ export default class ScreenController extends Component {
     }
 
     runtime.onOpenQrUrl = processOpenQrUrl
+    runtime.onWebxdcSendFile = (file, text) => {
+      ;(this.openDialog as OpenDialogFunctionType)(WebxdcSaveToChatDialog, {
```

Comment:
also here, consider naming things closer to what it does: WebxdcSendToChatDialog